### ISSUE TITLE
Global CLI packages

### ISSUE THREAD

**Issue by adamralph** [scriptcs/scriptcs#607]
For creating CLI tools, similar to global modules in node, e.g. grunt-cli.

Must have the ability to add commands to PATH.

**Comment by glennblock** [comment]
@adamralph do you want this?

**Comment by adamralph** [comment]
Definitely. It's a missing part of the ecosystem when you compare with node.

**Comment by glennblock** [comment]
Ahh right, agree. We need to think about how it work world.
